src/cirrus/docker.py

```diff
@@ -35,17 +37,15 @@ def __init__(self, cli_opts, config):
         self['login'] = config.get_param(
             'docker', 'docker_login_username', None
         ) is not None
-        self['tag'] = None
 
         if cli_opts.login:
             self['login'] = True
-        if hasattr(cli_opts, 'tag'):
-            if cli_opts.tag:
-                self['tag'] = cli_opts.tag
```

Comment:
Whats the use case for overriding the tag to not be the same as the cirrus tag?
I guess if you wanted to do something like X.Y.Z-beta on the image, but IMO you should just do minor versions for that